### ISSUE TITLE
adaptation of https://github.com/freifunkMUC/nodelist

### ISSUE THREAD

**Issue by Adorfer** [plumpudding/hopglass-server#10]
I would like to have a table of data available for the hopglas viewer, aggregated like the one coming out of 
https://github.com/freifunkMUC/nodelist

**Comment by anoymouserver** [comment]
I hope, that I haven't misunderstood your question, but the nodelist of ffmuc uses only the nodes.json and graph.json, which is also outputted by the Hopglass-Server. Accordingly, the nodelist is independent of the ffmap backend and should also work with this backend solution.

**Comment by plumpudding** [comment]
Yes it would. This ticket is mainly for both the server and the viewer. The idea is to integrate it into the viewer.
